wiki/Configuration:-Debug-Options.md

```diff
@@ -258,6 +259,22 @@ debug {
 }
 ```
 
+
+### 'wait-for-gpu-sync-in-screencasting'
+
+<sup>Since: next release</sup>
+
+Allows niri to wait for full GPU completion of each frame before handing it to PipeWire.
+
+Eliminates screen tearing in screencasts but may increase latency.
```

Comment:
```suggestion
Wait until every screencast frame is done rendering before handing it over to PipeWire.

Sometimes helps on NVIDIA to prevent glitched frames when screencasting.
```